### PR TITLE
Relax sonata/block-bundle dependency

### PR DESCRIPTION
**Pull request by EmmanuelVella**
Allow to install ``sonata/block-bundle`` 2.3 / 2.4

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "symfony/framework-bundle": "~2.3",
         "doctrine/phpcr-bundle" :"~1.0",
         "doctrine/phpcr-odm": "~1.0",
-        "sonata-project/block-bundle": "2.3.*",
+        "sonata-project/block-bundle": "~2.3",
```
Comment (dbu) [inline composer.json]: sonata was not following semver, doing BC breaks between minor versions. if you tested things to work with block bundle 2.4, we could do ">=2.3.0, <2.5.x-dev"

Comment (WouterJ) [inline composer.json]: Or `2.3.* | 2.4.*` :) (which I always find more easy to understand)

**Comment by EmmanuelVella** [comment]
I have updated my PR.

**Comment by dbu** [comment]
thanks!

**Pull request merged**